ipaserver/plugins/baseuser.py

```diff
@@ -157,6 +158,18 @@ def update_samba_attrs(ldap, dn, entry_attrs, **options):
                 )
 
 
+def validate_fido2key(ugettext, key):
+    """
+    Validate the format for fido2 key mappings.
+
+    The expected format is FIDO2:<key id>,<pubkey>
+    """
+    pattern = re.compile('FIDO2:.*,.*')
```

Comment:
This is a very broad regex. Are there at least length requirements we can impose on the key id, for example?

Reply:
SSSD team is considering returning the key as FIDO2:<key id>,<pem format> (https://github.com/SSSD/sssd/pull/6379).
When their PR is merged, IPA code can add a validation method.

Reply:
Minor note: s/FIDO2/PASSKEY

Reply:
The PR is merged, so you should be able to update the validation.

Reply:
@ikerexxe thanks for thé heads up.
PR updated with a check for the format (ensure the key part can be read as as PEM)